### ISSUE TITLE
Loinc 20636-7:  "Alanine [Moles/volume] in Serum or Plasma"

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/human-phenotype-ontology#4110]
Suggest creating a new term for Loinc: 20636-7 [Alanine [Moles/volume] in Serum or Plasma] 
New term label: Hypoalaninemia
New term comment (if any): Parent term: Abnormality of alanine metabolism
Your biocurator ID for loinc2hpo (if desired): JGM:lccarmody

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0500154
